build-farm/make-adopt-build-farm.sh

```diff
@@ -55,10 +55,19 @@ if [ -z "$JAVA_TO_BUILD" ]; then
   fi
 fi
 
-[ -z "$JAVA_TO_BUILD" ] && echo JAVA_TO_BUILD not defined - set to e.g. jdk8u && SANEVARS=1
+[ -z "$JAVA_TO_BUILD" ] && echo JAVA_TO_BUILD not defined - set to e.g. jdk8u
 [ -z "$VARIANT"       ] && echo VARIANT not defined - assuming hotspot && export VARIANT=hotspot
 [ -z "$FILENAME"      ] && echo FILENAME not defined - assuming "${JAVA_TO_BUILD}-${VARIANT}.tar.gz" && export FILENAME="${JAVA_TO_BUILD}-${VARIANT}.tar.gz"
 
+# shellcheck source=sbin/common/constants.sh
+source "$PLATFORM_SCRIPT_DIR/../sbin/common/constants.sh"
+
+# Check that the given variant is in our list of common variants
+if [ -z "$(echo "$BUILD_VARIANTS" | grep -w "$VARIANT")" ]; then
```

Comment:
The quotes inside another set of quotes looks a bit odd, but I presume it's been tested to work inside the `$()`

Reply:
This was mostly because shellcheck complained about them not being in quotes. It still picks up an invalid VARIANT but I can look into removing them or disabling the shellcheck rule if it's annoying to see?

Reply:
No I'm not overly concerned about it - happy for it to be left unless the next reviewer objects :-)
Thanks for the other changes though!

Reply:
It has already been implemented 🙂